src/api/spec/models/obs_factory/staging_project_spec.rb

```diff
@@ -1,10 +1,46 @@
 require 'rails_helper'
+require 'webmock/rspec'
 
 RSpec.describe ObsFactory::StagingProject do
-  let(:distribution) { ObsFactory::Distribution.new(create(:project, name: 'openSUSE:Factory')) }
+  let(:factory) { create(:project, name: 'openSUSE:Factory') }
+  let(:factoryppc) { create(:project, name: 'openSUSE:Factory:PowerPC') }
+  let!(:staging_a) { create(:project, name: 'openSUSE:Factory:Staging:A') }
+  let!(:staging_a_dvd) { create(:project, name: 'openSUSE:Factory:Staging:A:DVD') }
+  let!(:staging_b) { create(:project, name: 'openSUSE:Factory:Staging:B') }
+  let!(:staging_b_dvd) { create(:project, name: 'openSUSE:Factory:Staging:B:DVD') }
+  let!(:staging_c) { create(:project, name: 'openSUSE:Factory:Staging:C') }
+  let!(:staging_c_dvd) { create(:project, name: 'openSUSE:Factory:Staging:C:DVD') }
+  let(:opensuse_132) { create(:project, name: 'openSUSE:13.2') }
+  let!(:opensuse_staging_a) { create(:project, name: 'openSUSE:13.2:Staging:A') }
+  let!(:opensuse_staging_b) { create(:project, name: 'openSUSE:13.2:Staging:B') }
+  let(:factory_distribution) { ObsFactory::Distribution.find(factory.name) }
+  let(:staging_project_a) { ObsFactory::StagingProject.new(project: staging_a, distribution: factory_distribution) }
+  let(:staging_project_adi) { ObsFactory::StagingProject.new(project: staging_adi, distribution: factory_distribution) }
+
+  describe '::for' do
+    context 'openSUSE:Factory' do
+      subject { ObsFactory::StagingProject.for(factory_distribution) }
+
+      it { expect(subject.count).to eq(3) }
```

Comment:
What is the significance of this number?

Reply:
hello?

Reply:
@hennevogel, because we have different kind of projects and this should contain only staging projects of factory. But if you prefer I can remove it.

Reply:
I find it superfluous...

Reply:
taking you already list the 3 you expect in the following line

Reply:
removed